ramp-database/ramp_database/tools/event.py

```diff
@@ -433,6 +433,48 @@ def get_event(session, event_name):
     return select_event_by_name(session, event_name)
 
 
+def get_cv_fold_by_event(session, event):
+    """Get ScoreType from the database
+
+    Parameters
+    ----------
+    session : :class:`sqlalchemy.orm.Session`
+        The session to directly perform the operation on the database.
+    event_name : str or None
+        The event class to query
+
+    Returns
+    -------
+    cv fold : : list of all cv folds of this event
+    """
+    return (session.query(CVFold)
+                   .filter(EventScoreType.event_id ==
+                           event.id)
+                   .all())
+
+
+def get_score_type_by_event(session, event):
```

Comment:
We have an issue open where we tout with Roman to revert creating these functions and instead create the query directly. It is not saving so much line of code and this is explicit enough to make the query